src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -59,6 +59,8 @@
     protected $startTime;
     protected $loadClassCache;
 
+    private $projectDir;
```

Comment:
Is there a reason this is `private` and not `protected` like `$rootDir`. I need to override this in my kernel and can't really do at runtime…